src/common/prometheus/prometheus.provider.ts

```diff
@@ -33,32 +33,38 @@ export const PrometheusRPCErrorsCounterProvider = makeCounterProvider({
 
 export const PrometheusAccountBalanceGaugeProvider = makeGaugeProvider({
   name: METRIC_ACCOUNT_BALANCE,
-  help: 'Account balance',
-  labelNames: ['address'],
+  help: 'Account balance on the given network',
+  labelNames: ['address', 'network'],
```

Comment:
it looks like we can use the default labels instead of passing them everywhere manually https://github.com/siimon/prom-client#default-labels-segmented-by-registry

Reply:
Made an attempt.